### ISSUE TITLE
deprecation warnings with recent pygobject

### ISSUE THREAD

**Issue by anntzer** [spantaleev/sftpman-gtk#7]
sftpman-gtk 1.0.1, pygobject 3.27.1

at startup:
```
/usr/lib/python3.6/site-packages/sftpman_gtk/gui.py:9: PyGIDeprecationWarning: Since version 3.11, calling threads_init is no longer needed. See: https://wiki.gnome.org/PyGObject/Threading
  GObject.threads_init()
/usr/lib/python3.6/site-packages/sftpman_gtk/gui.py:270: PyGIDeprecationWarning: GObject.idle_add is deprecated; use GLib.idle_add instead
  GObject.idle_add(self.refresh_list)
```

**Comment by spantaleev** [comment]
Thanks for the heads-up!

I'm still on `pygobject = 3.26.1`, so I can't reproduce, but I've changed the code a bit to avoid the deprecation. It should be fine now.

Released as `sftpman-gtk = 1.0.2`

**Comment by anntzer** [comment]
thanks for the quick reply
